docs/commands.md

```diff
@@ -100,6 +100,10 @@ Skip dependencies installation
 
 Force use of npm during initialization
 
+#### `--bun`
```

Comment:
We should do `--package-manager=bun|npm|yarn` or `-pm=bun|npm|yarn`

Reply:
I thought about it, but it's so long to type :D We can have both as well as input and then convert to internal representation that bridges those

Reply:
additionally this would mean deprecation of `--npm` flag probably. Anyway people will probably just use `bunx react-native init` without using any additional flag. However if you think creating a flag to specify the package manager would be useful, I can do that. I can see the advantage of that solution once there is any new hot package manager available to use

Reply:
added `--pm` flag and set `--npm` as deprecated ✅